Header.php

```diff
@@ -30,7 +30,8 @@
 <link href="css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
 
 <!--<link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css"/> -->
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/fontawesome.min.css" />
+<!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/fontawesome.min.css" />-->
```

Comment:
@dzr201732120115 

Do we still need this line, `<!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/fontawesome.min.css" />-->`, since it looks like being commented out?

**Hui**

Reply:
I have deleted <!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/fontawesome.min.css" />-->

And link it to localization. 
<link rel="stylesheet" href="http://118.25.96.118/nor/font-awesome/css/font-awesome.min.css" />

Reply:
另外LRR网站中附带的font-awesome.min.css版本是4.0，Header.php去cdnjs中引用的是5.8.2，不过影响应该不大